DCC.combined-schema.json

```diff
@@ -264,7 +282,7 @@
           "$ref": "#/$defs/disease-agent-targeted"
         },
         "fr": {
-          "description": "ISO 8601 Date of First Positive NAA Test Result",
+          "description": "ISO 8601 complete date of first positive NAA test result",
```

Comment:
@gabywh Instead of "complete date” I'd prefer “full-date” as was used earlier for `v/dt`. That's also more in-line with this: https://datatracker.ietf.org/doc/html/rfc3339#section-5.6
(Also for 2 occurrences below + the ones in the combined schema.)

Reply:
> @gabywh Instead of "ISO 8601 complete date” I'd prefer “ISO8601 full-date” 

well, nice you give your preferences, but it's not a restaurant ;)

> as was used earlier for `v/dt`. That's also more in-line with this: https://datatracker.ietf.org/doc/html/rfc3339#section-5.6

but not inline with the formally approved documentation for v1.3.0 for dob:

> Complete or partial date without time restricted to the range from 1900-01-01 to 2099-12-31.

So it stays as "complete" to match the approved documentation

Reply:
The specs for `dob` and `v/dt` are completely different in this schema (patterned vs. string with date format), so I don't see the need to match those up. This comment is also specifically *not* about `dob`, but about the inconsistency of the documentation of the properties in a `t` having type string with date format.

Reply:
> The specs for `dob` and `v/dt` are completely different in this schema (patterned vs. string with date format), 

that's because they have very different constraints

> so I don't see the need to match those up. 

You mean `dob` and `v/dt`? No, no point, they have very different constraints.

> This comment is also specifically _not_ about `dob`, but about the inconsistency of the documentation of the properties in a `t` having type string with date format.

Which documentation? Concrete examples?

Reply:
Documentation = `description` fields in the schema.

My comment does not pertain at all to the `dob` field, just to all fields of type string with `format: date`. At least the descriptions are consistent across the `recovery_entry` type, so OK to leave as-is.

Reply:
> My comment does not pertain at all to the `dob` field, 

yes, already understood

> just to all fields of type string with `format: date`. 

concrete examples? 
Or do you mean the * one * case of description for `v/dt` not being consistent with description for `r/df` and `r/du` ?

> At least the descriptions are consistent across the `recovery_entry` type, so OK to leave as-is.

So what was the review point ?